ppr-ui/src/layouts/LayoutUser.vue

```diff
@@ -9,20 +9,17 @@
     sbc-footer
 </template>
 
-<script>
-    import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
-    import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
+<script lang="ts">
+  import {createComponent} from "@vue/composition-api";
+  import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
+  import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 
-    export default {
-        name: 'LayoutUser',
-        components: {
-            SbcHeader,
-            SbcFooter
-        },
-        data: function () {
-            return {
-            }
-        },
-        inject: ['authApiUrl', 'originUrl'],
-    }
+  export default createComponent({
+    name: 'LayoutUser',
+    components: {
+      SbcHeader,
+      SbcFooter
+    },
+    inject: ['authApiUrl', 'originUrl'],
+  })
```

Comment:
Note that this doesn't use `setup()`.  I didn't see the value here as it would only be a more complex way of handling the `inject` portion of the component, in this case.  That's one of the nice things, is that the old-school way of building components still works.